### PR TITLE
MaxLines property on Text component

### PR DESCRIPTION
**Pull request by ichan-mb**
Added `MaxLines` property to the `Text` component to limit the number of lines to display when `TextWrapping` property is set to `Wrap`

**Examples**:
```
<ClientPanel>
	StackPanel ItemSpacing="10">
		<Text Value="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." />
		<Text Value="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." TextWrapping="Wrap" />
		<Text Value="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum." MaxLines="3" TextWrapping="Wrap" Color="Red" />
	</StackPanel>
</CllientPanel>
```

This PR contains:
- [x] Changelog
- [x] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Text/Wrap.uno [hunk 1]
```diff
@@ -42,9 +43,24 @@ namespace Fuse.Text
...
+			{
+				result = result.Take(maxLines).ToList();
+				// adding `...`
+				InsertTruncationSringSymbol(font, result);
+				return result;
+			}
 			return result;
 		}
 
+		static void InsertTruncationSringSymbol(Font font, List<List<ShapedRun>> lines)
```
Comment (mortend) [inline Source/Fuse.Text/Wrap.uno:56]: Typo: Missing a `t` in `InsertTruncationSringSymbol`.

Comment (ichan-mb) [reply Source/Fuse.Text/Wrap.uno]: oops! fixed.

Comment (mortend) [reply Source/Fuse.Text/Wrap.uno]: Thank you!

**Review by mortend — APPROVED** [review]

**Comment by mortend** [comment]
- [x] https://travis-ci.org/github/fuse-open/fuselibs/builds/749752007

**Pull request merged**